de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5471440_sys_gh2340_ReferenceTarget_In_RelationType.sql

```diff
@@ -0,0 +1,72 @@
+-- 2017-09-12T14:39:40.248
+-- URL zum Konzept
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,543415,0,'IsReferenceTarget',TO_TIMESTAMP('2017-09-12 14:39:40','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','IsReferenceTarget','IsReferenceTarget',TO_TIMESTAMP('2017-09-12 14:39:40','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
Please add a help test about what this new flag is about.
Also, please mention the github URL

Reply:
Done